CRM/Event/Form/ManageEvent/EventInfo.php

```diff
@@ -211,13 +211,8 @@ public static function formRule($values) {
     $errors = array();
 
     if (!$values['is_template']) {
-      if (empty($values['start_date'])) {
-        $errors['start_date'] = ts('Start Date and Time are required fields');
-      }
-      else {
-        if (($values['end_date'] < $values['start_date']) && !empty($values['end_date'])) {
-          $errors['end_date'] = ts('End date should be after Start date.');
-        }
+      if (($values['end_date'] < $values['start_date']) && !empty($values['end_date'])) {
```

Comment:
@JKingsnorth a small change if possible. This line might throw notice error if end_date is not set in $values. So can we replace this line with

`if (!empty($values['end_date']) && ($values['end_date'] < $values['start_date'])) {`

Thanks
Pradeep

Reply:
@pradpnayak - lol, I was actually just making that change while you were typing that. I've pushed up the new version with commits squashed down. ;p

Why is it that the 'simple' changes never turn out to be that simple?